test/desktop/features/scripts/build_maze_runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+export PATH="/Applications/Unity/Hub/Editor/$UNITY_VERSION/Unity.app/Contents/MacOS:$PATH"
+
+# Set project_path to the repo root
+pushd "${0%/*}"
+  pushd ../../../..
+    package_path=`pwd`
+    echo "Expecting to find Busnag package in: $package_path"
+  popd
+  pushd ../fixtures
+    log_file="$package_path/unity.log"
+    project_path="$(pwd)/maze_runner"
+
+    echo "Importing $package_path/Bugsnag.unitypackage into $project_path"
+    Unity -nographics -quit -batchmode \
+      -logFile $log_file \
+      -projectPath $project_path \
+      -ignoreCompilerErrors \
+      -importPackage "$package_path/Bugsnag.unitypackage"
+    RESULT=$?
+    if [ $RESULT -ne 0 ]; then exit $RESULT; fi
+
+    cp -R NativeCrashy.bundle maze_runner/Assets/Plugins/OSX/NativeCrashy.bundle
```

Comment:
Is there a reason this file can't already exist at that location?

Reply:
You're right - I've `git mv` moved into place and removed the line from the script.  It means building the test fixture manually with the editor is a more intuitive, 2-step, process - import the notifier and build.